cl_dll/hud_watermark.cpp

```diff
@@ -35,11 +35,14 @@ int CHudWatermark::Draw(float time)
 	int r, g, b;
 	UnpackRGB(r, g, b, gHUD.m_iDefaultHUDColor);
 
-	gEngfuncs.pfnDrawString(ScreenWidth / 20, gHUD.m_scrinfo.iCharHeight, "OpenAG client build " __DATE__, r, g, b);
-	gEngfuncs.pfnDrawString(ScreenWidth / 20, gHUD.m_scrinfo.iCharHeight * 2, "j.mp/OpenAG", r, g, b);
-
-	if (update_is_available)
-		gEngfuncs.pfnDrawString(ScreenWidth / 20, gHUD.m_scrinfo.iCharHeight / 2 * 7, "An update is available.", r, g, b);
-
+	extern cvar_t* hud_watermark;
+	if (hud_watermark->value == 1)
+	{
+		gEngfuncs.pfnDrawString(ScreenWidth / 20, gHUD.m_scrinfo.iCharHeight, "OpenAG client build " __DATE__, r, g, b);
+		gEngfuncs.pfnDrawString(ScreenWidth / 20, gHUD.m_scrinfo.iCharHeight * 2, "j.mp/OpenAG", r, g, b);
+
+		if (update_is_available)
```

Comment:
Maybe if an update is available, then this part of the HUD should be shown regardless of hud_watermark? What do you think?

Reply:
> Maybe if an update is available, then this part of the HUD should be shown regardless of hud_watermark? What do you think?

Perhaps the update disclaimer can be shown for a shorter time if hud_watermark == 0.

Reply:
Sounds good

Reply:
Changes have been pushed